apps/public-docsite-v9/src/Concepts/Poster/metadata.ts

```diff
@@ -0,0 +1,303 @@
+import { LibraryInfo } from './types';
+
+export const libraryInfo: LibraryInfo = {
+  name: 'Fluent UI React v9',
+  areas: [
+    {
+      name: 'Basic Inputs',
+      packages: [
+        {
+          name: 'react-button',
+          codeItems: [
+            {
+              name: 'Button',
+              componentType: 'component',
+            },
+            {
+              name: 'CompoundButton',
+              componentType: 'component',
+            },
+            {
+              name: 'MenuButton',
+              componentType: 'component',
+            },
+            {
+              name: 'SplitButton',
+              componentType: 'component',
+            },
+            {
+              name: 'ToggleButton',
+              componentType: 'component',
+            },
+          ],
+        },
+        {
+          name: 'react-checkbox',
+          newColumn: true,
+          codeItems: [{ name: 'CheckBox', componentType: 'component' }],
+        },
+        {
+          name: 'react-input',
+          codeItems: [{ name: 'Input', componentType: 'component' }],
+        },
+        {
+          name: 'react-label',
+          codeItems: [{ name: 'Label', componentType: 'component' }],
+        },
+        {
+          name: 'react-link',
+          newColumn: true,
+          codeItems: [{ name: 'Link', componentType: 'component' }],
+        },
+        {
+          name: 'react-radio',
+          codeItems: [
+            { name: 'Radio', componentType: 'component' },
+            { name: 'RadioGroup', componentType: 'component' },
+          ],
+        },
+        {
+          name: 'react-select',
+          codeItems: [{ name: 'Select', componentType: 'component' }],
+        },
+        {
+          name: 'react-slider',
+          newColumn: true,
+          codeItems: [{ name: 'Slider', componentType: 'component' }],
+        },
+        {
+          name: 'react-spinbutton',
+          codeItems: [{ name: 'SpinButton', componentType: 'component' }],
+        },
+        {
+          name: 'react-textarea',
+          codeItems: [{ name: 'TextArea', componentType: 'component' }],
+        },
+        {
+          name: 'react-switch',
+          codeItems: [
+            {
+              name: 'Switch',
+              componentType: 'component',
+            },
+          ],
+        },
+      ],
+    },
+    {
+      name: 'Menus and Navigation',
+      packages: [
+        {
+          name: 'react-accordion',
+          codeItems: [
+            { name: 'Accordion', componentType: 'component' },
+            { name: 'AccordionContext', componentType: 'component' },
```

Comment:
This will need to be updated after the [breaking change PR](https://github.com/microsoft/fluentui/pull/23092/files#diff-22e71f11d9c9660a4668b396542668d9a763dfdb6577dc153ee3b1dba107cce7) is merged as we're no longer exporting these contexts.